solr/solr-ref-guide/modules/configuration-guide/pages/package-manager-internals.adoc

```diff
@@ -115,7 +115,7 @@ openssl dgst -sha1 -sign my_key.pem runtimelibs.jar | openssl enc -base64 | sed
 +
 [source, bash]
 ----
-curl --data-binary @runtimelibs.jar -X PUT  http://localhost:8983/api/cluster/files/mypkg/1.0/myplugins.jar?sig=<signature-of-jar>
+curl --data-binary @runtimelibs.jar -X PUT  http://localhost:8983/api/cluster/filestore/files/mypkg/1.0/myplugins.jar?sig=<signature-of-jar>
```

Comment:
maybe drops the `files`?

Reply:
(See other comments - "files" is our way to distinguish between "metadata" and "commands", which are both under the "/filestore" path.)